packages/ai/src/ai/modules/task/commands/cmd_billing.py

```
@@ -0,0 +1,153 @@
+    Attributes:
+        _server: Reference to the TaskServer for context
+        connection_id: Unique identifier for this DAP connection
+        transport: Underlying transport mechanism for DAP communication
+    """
+
+    def __init__(
+        self,
+        connection_id: int,
+        server: 'TaskServer',
+        transport: TransportBase,
+        **kwargs,
```

Ruff (0.15.10)

[warning] Missing type annotation for `**kwargs`

(ANN003)